### DISCUSSION TITLE
How to assign crazyflies to different radios?

### DISCUSSION THREAD

**Discussion by girayus** [USC-ACTLab/crazyswarm#537] [Q&A] [answered]
Hello,

I have 6 crazyflies and 2 radios, and I want to assign 3 crazyflies per radio since I use cmdVelocityWorld commands in a 20 Hertz loop. What I understand from https://crazyswarm.readthedocs.io/en/latest/configuration.html#set-up-radio-communication is different channels mean different radios, i.e. for example yaml file

crazyflies:
  - id: 1
    channel: 70
    initialPosition: [1.75, 0.75, 0.0]
    type: default
  - id: 2
    channel: 70
    initialPosition: [1.0, 0.0, 0.0]
    type: default
  - id: 3
    channel: 70
    initialPosition: [1.75, 0.0, 0.0]
    type: default
  - id: 4
    channel: 80
    initialPosition: [2.5, 0.0, 0.0]
    type: default
  - id: 5
    channel: 80
    initialPosition: [1.25, -0.75, 0.0]
    type: default
  - id: 6
    channel: 80
    initialPosition: [2.0, -0.75, 0.0]
    type: default

first 3 cfs are assigned to radio 1, channel 70 and other 3 are assigned to radio 2, channel 80. Is this true? If it is not, how can I assign crazyflies to different radios?

**Comment by whoenig** [comment, marked as answer]
This is correct. The order you list the crazyflies in does not matter, i.e., you can "mix" channels there. There will be an exception if you have not enough radios plugged in.
